gustavo.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Gustavo Arsenio's talk blog ",
+    "repo": "https://github.com/GustavoArsenio/GustavoArsenio.github.io",
+    "owner": {
+        "username": "GustavoArsenio",
+        "email": "",
+        "twitter": ""
```

Comment:
Please add either an email or a twitter account. We need a way to contact you in case we need to inform you of any changes.